hpx/lcos/future.hpp

```diff
@@ -14,43 +14,36 @@
 #include <hpx/assertion.hpp>
 #include <hpx/concepts/concepts.hpp>
 #include <hpx/errors.hpp>
-#include <hpx/functional/bind.hpp>
-#include <hpx/functional/function.hpp>
 #include <hpx/functional/invoke.hpp>
-#include <hpx/functional/result_of.hpp>
 #include <hpx/functional/traits/is_callable.hpp>
 #include <hpx/lcos/detail/future_data.hpp>
-#include <hpx/lcos/detail/future_traits.hpp>
-#include <hpx/lcos_fwd.hpp>
 #include <hpx/memory/intrusive_ptr.hpp>
-#include <hpx/runtime/launch_policy.hpp>
 #include <hpx/serialization/detail/polymorphic_nonintrusive_factory.hpp>
 #include <hpx/timing/steady_clock.hpp>
 #include <hpx/traits/acquire_shared_state.hpp>
 #include <hpx/traits/future_access.hpp>
 #include <hpx/traits/future_then_result.hpp>
 #include <hpx/traits/future_traits.hpp>
-#include <hpx/execution/traits/is_executor.hpp>
 #include <hpx/traits/is_future.hpp>
-#include <hpx/traits/is_launch_policy.hpp>
-#include <hpx/type_support/always_void.hpp>
 #include <hpx/type_support/decay.hpp>
-#include <hpx/type_support/identity.hpp>
-#include <hpx/type_support/lazy_enable_if.hpp>
-#include <hpx/type_support/void_guard.hpp>
-#include <hpx/util/serialize_exception.hpp>
```

Comment:
I think you'll need to add back this line (`serialize_exception.hpp`). The pycicle builder that's failing is running with networking off. The builders with networking on are likely getting the header in the right place by mistake, but the header is needed in here.

Reply:
Ok thanks !